PWGHF/HFL/Tasks/taskSingleMuon.cxx

```diff
@@ -180,6 +181,8 @@ struct HfTaskSingleMuonSelection {
     registry.add("hEtaAcuts", "", hTHnEta);
     registry.add("hPBcuts", "", hTHnP);
     registry.add("hPAcuts", "", hTHnP);
+    registry.add("hMuLiteBcuts", "", hTHnMuLite);
+    registry.add("hMuLiteAcuts", "", hTHnMuLite);
```

Comment:
You might want to consider adding only histograms that are needed based on the activated process functions.